### ISSUE TITLE
No domain in sitemap and google preview

### ISSUE THREAD

**Issue by nebrot** [clickstorm/cs_seo#42]
We are using cs_seo 1.8.5 with realurl 2.1.5 in TYPO3 8.5.1 and created a domain record at root page, but the domain is missing in preview and sitemap:

![seo-domain](https://cloud.githubusercontent.com/assets/6941112/22144155/094b7a94-defd-11e6-8e5a-9f53b0388d85.png)

![seo-sitemap](https://cloud.githubusercontent.com/assets/6941112/22144143/fef3e46e-defc-11e6-95c0-1e10cf36f572.png)

![seo-preview](https://cloud.githubusercontent.com/assets/6941112/22144152/070a2ac8-defd-11e6-8c5d-7e3e3734de6d.png)

Our typoscript:
```
plugin.tx_csseo.sitemap {
	pages {
		rootPid = 1
		languageUids = 0
	}
}

lib.currentUrl.typolink.additionalParams.wrap = 
```

Ext.-Config is also `basic.tsConfigPid = 1`
We are using `config.absRefPrefix = /` and no `baseURL`.

What could be wrong?

**Comment by mhirdes** [comment]
Hi, you have to use config.absRefPrefix = <yourProtocoll>://<yourDomain>/ as it shown in the RealURL doc https://wiki.typo3.org/Realurl/manual . To fix the L=0 you maybe could send me your RealURL config file or we take together a look in your backend.

**Comment by nebrot** [comment]
Thanks for your fast help!

With setting `config.absRefPrefix = http://example.com/` sitemap is now correct.
Would it be possible that it will also work only with `/`? Because we use this in our TYPO3-Dummy for new homepages for our customers. With only `/` it's compatible to any domain and protocol (http/https) and one less source of error in configuration.
With [dd_googlesitemap-extension](https://github.com/dmitryd/typo3-dd_googlesitemap/blob/development/Classes/Generator/PagesSitemapGenerator.php) it works with only `/` too.

I truncated realurl-tables and now it's working, thank you anyway :-)

**Comment by mhirdes** [comment]
Hi, I tested this behaviour now. If I set 

`config.absRefPrefix = /
config.baseURL >`

and delete the realURL cache everything looks ok. Also without any domain record.

**Comment by mhirdes** [comment]
I can only offer you to check your TYPO3 together via TeamViewer or you give me a backend account. If you want to, let me know per mail hirdes@clickstorm.de. Until then I will close this issue.

**Comment by mhirdes** [comment]
Found the bug. It is core related in TYPO3 8.5. 

I use the uri.page viewhelper and say absolute:1. 
In sysext/extbase/Classes/Mvc/Web/Routing/UriBuilder.php there is an if statement if($absolute===true) but here (absolute===1) and so it failse. But I can't set inline in fluid an variable to true.

https://forge.typo3.org/issues/79375 

It should be fixed in TYPO3 8.6.

**Comment by nebrot** [comment]
Thanks for your help! I'll try TYPO3 8.6 after release next week and let you know if it's working.

**Comment by nebrot** [comment]
They fixed it in TYPO3 8.6, thank you for your help and this great extension!
